docs/catalog/cloud/reduce-network-traversal-between-VMs.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- network
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Reduce network traversal between VMs
+
+## Description
+Placing VMs in a single region or a single availability zone reduces the physical distance between the instances. 
+
+However, for business critical workloads, you need to ensure your workload is spread across multiple availability-zones, which may result in more network traversal and increase in your carbon footprint.
+
+## Solution
+Consider using proximity placement groups (also called VM-Host affinitry rules) to reduce the network traversal by ensuring your VM resources are physically located close to each other. 
```

Comment:
Are proximity placement groups available outside of Azure?  Patterns should be vendor-agnostic so I would look to make this a more generic solution that is not Azure-specific.